### DISCUSSION TITLE
Welcome to mzmine3 Discussions!

### DISCUSSION THREAD

**Discussion by robinschmid** [mzmine/mzmine#1024] [Announcements]
## 👋 Hey community
 We will be testing the discussions here as another way to communicate MZmine-related things like developments, ideas, and collaborations. Feel free to post MZmine or mass spec-related topics here.

Issues should go into the issues section.

**Comment by zglong1** [comment]
Anyone who's used the GC-MS peak picking/deconvolution/export for library search (ie. GNPS, etc.), any tips? I feel like I've followed the MZMine and/or GNPS workflows pretty diligently and seem to have found settings/parameters appropriate for my instrument/data, but seem to be struggling with getting consistent fragmentation spectra between technical replicates of samples (that otherwise look expectedly similar in the TIC). Any guidance available somewhere on refining parameters or can someone take a look at my project file and give some suggestions? Right now I ran a practice set of three derivatization blanks and three technical replicates, but the main issue is the lack of consistent peak detection across samples (discounting the peaks which should only be in the real samples). Going through some of the papers linked in the documentation, I feel like I have to be doing something dumb.

**Reply by jayram1988** [reply]
Dear Mzmine team,
I am trying to upload the mzML file in the Mzmine3, but  i am getting error msg as given below:
Error parsing mzML:java.lang.NumberFormatException:For input string: "nan" (floatingDecimal.Java.2054)
Please help me in this regard..
Thanks
Jayram

**Comment by waterflow80** [comment]
Hello,
I am software engineering student and I am really willing to contribute to MZmine.
I will be happy if can find some little issues where I can contribute to MZmine, so I can get 
familiarized with the code structure and be able to do some more advanced and complex
contributions in the future.
I'll be very glad to work with you guys.
